xhr.bs

```diff
@@ -1039,18 +1039,20 @@ method must run these steps:
  <a for=body>total bytes</a>.
 
  <li><p>If the <a>synchronous flag</a> is unset, <a>fire a progress event</a> named
- <a event><code>progress</code></a> with <var>transmitted</var> and <var>length</var>.
+ <a event><code>progress</code></a> at <b>this</b> with <var>transmitted</var> and <var>length</var>.
 
  <li><p>Set <a>state</a> to <i>done</i>.
 
  <li><p>Unset the <a><code>send()</code> flag</a>.
 
- <li><p><a>Fire an event</a> named <a event><code>readystatechange</code></a>.
+ <li><p><a>Fire an event</a> named <a event><code>readystatechange</code></a> at <b>this</b>.
 
  <li><p><a>Fire a progress event</a> named <a event><code>load</code></a>
+ at <b>this</b>
  with <var>transmitted</var> and <var>length</var>.
 
  <li><p><a>Fire a progress event</a> named <a event><code>loadend</code></a>
+ at <b>this</b>'s {{XMLHttpRequestUpload}} object
```

Comment:
This is wrong, this should be at **this** only.

Reply:
Got it